layouts/index.html

```diff
@@ -132,14 +132,13 @@ <h2><i class="fa-light fa-microchip-ai"></i> Faster scikit-learn <br> with cuML<
                 </div>
                 <div class="col-md-4 py-3">
                     <h2><i class="fa-light fa-circle-nodes"></i> Faster NetworkX <br> with cuGraph</h2>
-                    <p>cuGraph makes migration from NetworkX easy, accelerates graph analytics, and allows scaling far
-                        beyond existing tools. </p>
-                    <a href="https://github.com/rapidsai/cugraph/blob/branch-24.10/notebooks/cugraph_benchmarks/synth_release_single_gpu.ipynb"
+                    <p>cuGraph accelerates NetworkX with zero code changes for much greater performance at scale.</p>
+                    <a href="https://github.com/rapidsai/cugraph/blob/branch-24.10/benchmarks/nx-cugraph/pytest-based"
                         target="_blank">Run this benchmark yourself <i class="fa-solid fa-arrow-up-right"></i> </a>
 
                     <canvas id="cuGraphChart" width="400" height="350"></canvas>
-                    <p class="fs-8 text-muted">* Benchmark on AMD EPYC 7642 (using 1x 2.3GHz CPU core) w/ 512GB and
-                        NVIDIA A100 80GB (1x GPU) w/ NetworkX v3.0 and cuGraph v23.02</p>
+                    <p class="fs-8 text-muted">* Benchmark on Intel(R) Xeon(R) w9-3495X w/ 250 GB and
```

Comment:
250 GB of RAM seems wrong. Is it 256 GB?

Reply:
I would think so, but I'm copying from the report received from the benchmark team, which is also what marketing published.
![image](https://github.com/user-attachments/assets/02d780be-823e-4d60-9972-a11478185bfe)